docs/source/user_manual/matplotlib_to_chaco.rst

```diff
@@ -0,0 +1,55 @@
+|                        |                                          |                   `p.value_range.high_setting = HIGH`                  |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+| Label lines for legend |       `ax.plot(…, label=label)`          |                        `p.plot(…, name=name)`                          |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|       Add legend       |              `ax.legend()`               |                       `p.legend.visible = True`                        |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|    Legend position     |      `ax.legend(loc='lower left')`       |                      `p.legend_alignment = 'll'`                       |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|    Background color    |        ax.set_facecolor('white')         |                          `p.bgcolor = 'white'`                         |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|      Show spines       |      ax.spines.set_color('black')        |                       `p.border_visible = True`                        |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|       Line width       |         `ax.plot(linewidth=lw)`          |           `p.line_width = 1.1` or `p.plot(…, linewidth=lw)`            |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|         X grid         |           `ax.grid(axis='x')`            |                       `p.x_axis.visible = True`                        |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|         Y grid         |           `ax.grid(axis='y')`            |                       `p.y_axis.visible = True`                        |
++------------------------+------------------------------------------+------------------------------------------------------------------------+
+|     Second y ticks     |  `ax.yaxis.set_label_position('right')`  |                   `p.y_axis.orientation = "right"`                     |
+|                        |         `ax.yaxis.tick_right()`          |                                                                        |
```

Comment:
having two entries in the "matplotlib" column here seems to mess with the formatting - the first entry is in bold as if it were a title.

Reply:
Any idea how to remedy this?